files/default/ckan_cli

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+# Call either 'ckan' or 'paster' with appropriate syntax,
+# depending on what is present.
+
+# This script is aware of the VIRTUAL_ENV environment variable, and will
+# attempt to respect it with similar behaviour to commands like 'pip'.
+# Eg placing this script in a virtualenv 'bin' directory will cause it
+# to call the 'ckan' or 'paster' command in that directory, while
+# placing this script elsewhere will cause it to rely on the VIRTUAL_ENV
+# variable.
+
+CKAN_INI_FILE="${CKAN_INI_FILE:-/etc/ckan/default/production.ini}"
+PASTER_PLUGIN="${PASTER_PLUGIN:-ckan}"
```

Comment:
Also need more information on this.  That its not required for 2.9 (ckan cli) it is required for ckan 2.8- via paster?